### PR TITLE
Dev/teampage PRA-156

### PR DESCRIPTION
**Pull request by ElenaGor8**
## One Line Description
Responsive Our Team page and footer

## Requirements

_What this code should do_

Our Team page shows 9 team members, their images and short "about" text, each member card is clickable to their personal github pages.
Footer should show in the bottom of the pages throughout the app.
footer icons click out to corresponding webpages.
footer nav buttons styled and properly routed.
Buttons changed to solid green in Homepage and Contact pages
Responsive design

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

See footer in laptop and mobile layouts, click on Meet the Team to see Our Team page, also check it in mobile version.

## UI/UX Outcome

_Attach a screenshot here_
OurTeammobile

OutTeam_Footer_mobile

OurTeam827

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaPestova1 — APPROVED** [review]
We have the well behave footer now! Great! I like all here

**Review by AnnaSolovykh — APPROVED** [review]
LGTM!

File: src/components/Layout/OurTeam.tsx [hunk 1]
```diff
@@ -1,14 +1,112 @@
...
+    imageUrl: "/images/AnnaSolovykh.jpeg",
+    bio: "A web developer specializing in Frontend (JS, React, Redux), wishing to improve and master.",
+    githubUrl: "https://github.com/AnnaSolovykh"
+  },
+  {
+    name: "Aigul Yedigeyeva",
+    role: "Back-end",
+    imageUrl: "/images/kitten.png",
+    bio: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    githubUrl: "https://github.com/AigulYe"
```
Comment (Ekaterina-Bondareva) [inline src/components/Layout/OurTeam.tsx:31]: Could you please fix the link. This link is not work. I suppose the correct one is: `https://github.com/AigulY`

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Great job! Love how the page looks like! Please fix one line of code (GitHub link).

**Review by rebekahcallkacz — COMMENTED** [review]
Not sure if you want to address this in this PR, but the spacing is off on the Add Recipe page. The footer is slightly hidden on my screen.

**Review by rebekahcallkacz — COMMENTED** [review]
I might use the term "Front End Developer" and "Back End Developer" (or Engineer instead of developer) instead of just "Front-end" and "Back-end". You usually put a person's "title" instead of just what area they're in.

You'll also want to italicize that title/role to separate it from names - see [this example](https://zelusanalytics.com/).

**Review by rebekahcallkacz — APPROVED** [review]
LGTM other than the link that Kate already pointed out.

**Comment by ElenaGor8** [comment]
Made changes to OurTeam page and Footer for responsiveness, added mentors' cards, adjusted height of the Add recipe page, corrected Aigul's gihub link

**Review by AnnaPestova1 — APPROVED** [review]

**Review by Ekaterina-Bondareva — APPROVED** [review]
LGTM! Good job!

**Pull request merged**